Package.swift

```diff
@@ -8,7 +8,7 @@ let package = Package(
     name: "SwiftGodotKick",
     products: [
         .executable(
-            name: "swift-godot-kick",
+            name: "CreateProject",
```

Comment:
Ah this was my mistake, the README should have been updated to `swift run swift-godot-kick`
Because, when putting this executable in the path `swift-godot-kick` is a much clearer name than `CreateProject`